src/js/pages/services/DeploymentsTab.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+import AlertPanel from '../../components/AlertPanel';
+
+var DeploymentsTab = React.createClass({
```

Comment:
No. I guess this is preferred? I'll open a PR with a documentation update if so.